src/containers/App.jsx

```diff
@@ -51,11 +55,18 @@ const router = createBrowserRouter([
 ]);
 
 const App = () => {
+  const [token, setToken] = useState(null);
+  const loginCurrentUser = (newUserData) => {
+    setToken(newUserData);
+  };
+
   return (
     <ErrorBoundary fallback={<NotFoundPage />}>
+      <AuthContext.Provider value={{ token, loginCurrentUser }}>
       <div className={styles.app}>
         <RouterProvider router={router} />
       </div>
+      </AuthContext.Provider>
     </ErrorBoundary>
   );
```

Comment:
can we remove `loginCurrentUser` and then put `setToken` into context directly ? 
and use `setToken` elsewhere the same way as we use `loginCurrentUser`...

Reply:
of course! it wasn't my piece of code

Reply:
Done

Reply:
👍  
it wasn't you piece of code, but luckily it is the task you are currently working on and you can make it better than it was before.

i hope, this is your actual philosophy ;)

Reply:
yes, of course:)